src/sentry/static/sentry/app/views/settings/organizationDeveloperSettings/sentryApplicationRowButtons.tsx

```diff
@@ -0,0 +1,223 @@
+import React from 'react';
+
+import styled from 'react-emotion';
+
+import Access from 'app/components/acl/access';
+import Button from 'app/components/button';
+import ConfirmDelete from 'app/components/confirmDelete';
+import Confirm from 'app/components/confirm';
+import space from 'app/styles/space';
+import {t} from 'app/locale';
+import {Organization, SentryApp, SentryAppInstallation} from 'app/types';
+
+type Props = {
+  organization: Organization;
```

Comment:
@nolachen We should start using `LightWeightOrganization` whenever we don't need the extra fields on an org. I realize this is how it was before but we didn't have that type until recently.

Reply:
@scefali Lol it is way harder than I realized to do this, because there are some dependencies that go all the way into getsentry. Think we might need to revisit this later in a different PR